darwin/main.m

```diff
@@ -3,6 +3,8 @@
 
 static BOOL canQuit = NO;
 static NSAutoreleasePool *globalPool;
+static applicationClass* app;
```

Comment:
We could get rid of `realNSApp()` with this. I only provide it because `NSApp` by default is `id`, which can confuse clang and generate garbage warnings like `setDelegate:` having the wrong protocol type.